test-app/types/ember-data/types/registries/model.d.ts

```diff
@@ -2,5 +2,6 @@
  * Catch-all for ember-data.
  */
 export default interface ModelRegistry {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
This is a file from the v2 generator / boilerplate, so I think we could actually remove this if we wanted to.  Without disabling this we were getting an error and I figured it's "fine" since it's in the test-app, but also have 0 strong opinions on flat out removing this file completely if we don't find any value in it.

Reply:
I think it's reasonable to remove. We will never deal with ember-data in this repo!